### PR TITLE
[Bug Fix] Silent recaptcha with challenge if confidence is low

### PR DESCRIPTION
**Pull request by alis-khadka**
Addresses: https://github.com/restarone/violet_rails/issues/916

### **Demo Video**

https://user-images.githubusercontent.com/25191509/182033120-061b7e9a-836c-4f72-8fe4-ed5da43083ac.mp4

### CODE REVIEW

**Comment by donrestarone** [comment]
@alis-khadka great first effort!

When the challenge is presented, the message should be "Please prove your humanity by solving the challenge": 

Screen Shot 2022-07-31 at 11 21 21 AM

File: Gemfile [hunk 1]
```diff
@@ -94,6 +94,7 @@ group :test do
   gem 'rails-controller-testing'
   gem 'mocha'
   gem 'webmock'
+  gem "minitest-stub_any_instance"
```
Comment (donrestarone) [inline Gemfile:97]: @alis-khadka no need to add this gem, we already have stubbing support via `mocha` (eg: test/controllers/simple_discussion/forum_threads_controller_test.rb)

**Comment by alis-khadka** [comment]
> Please prove your humanity by solving the challenge

You mean - we should add another **flash message** saying: "Please prove your humanity by solving the challenge".

Or, should we add that text just above the v2 challenge box like below?
image

**Comment by donrestarone** [comment]
> > Please prove your humanity by solving the challenge
> 
> You mean - we should add another **flash message** saying: "Please prove your humanity by solving the challenge".
> 
> Or, should we add that text just above the v2 challenge box like below? image

@alis-khadka exactly how you have it is good

**Pull request merged**